### PR TITLE
Setup Firebase Analytics for Eatery

### PR DESCRIPTION
**Pull request by kevinchan159**
Issue: #200 
Overview: This PR does setup to incorporate Firebase Analytics for Eatery. It also adds a new `AppDevAnalytics` cocoapod which contains some nice classes to use when logging.

Test: Made sure app runs & confirmed that everything is setup in Firebase

`NOTE:` I will post the `GoogleService-Info.plist in #eatery-ios`. You will need this for Firebase to run properly

### CODE REVIEW

File: Eatery.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -888,7 +896,7 @@
 				60FF4E1C1CBEE03E0051FD01 /* MapViewController.swift in Sources */,
 				D956DCC12187B94E001BB9B0 /* Event.swift in Sources */,
 				6502AF871DCFB9ED00390EAA /* FilterBar.swift in Sources */,
-				D10246301C7A320C000D5CD8 /* (null) in Sources */,
```
Comment (Omarrasheed) [inline Eatery.xcodeproj/project.pbxproj]: same here

**Review by Omarrasheed — APPROVED** [review]
LGTM

**Review by whoiswillma — APPROVED** [review]
Looks good 👍

File: Eatery/AppDelegate.swift [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import UIKit
-import SwiftyJSON
-import Fabric
 import Crashlytics
+import Fabric
```
Comment (chalo2000) [inline Eatery/AppDelegate.swift:2]: If we have completed the setup for FireBase Analytics, is it still necessary to import Fabric or is the plan to keep it until Google deprecates?

**Review by chalo2000 — COMMENTED** [review]
LGTM

**Pull request merged**